x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -425,7 +425,7 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
   return (
     <EuiFlyout
       ownFocus
-      maskProps={{ style: 'z-index: 5000' }} // For an edge case to display above the timeline flyout
+      maskProps={{ style: 'z-index: 1000' }} // For an edge case to display above the timeline flyout
```

Comment:
removing this line for testing the PR on Cloud env, then will revert it back until we fix it

Reply:
reverted back https://github.com/elastic/kibana/pull/146750/commits/208a29e9e85bd7477bbbfa7925028bd353cb9089 after testing